src/binary-reader-objdump.c

```diff
@@ -265,6 +270,22 @@ static WasmResult on_opcode_uint64(WasmBinaryReaderContext* ctx,
   return WASM_OK;
 }
 
+static WasmResult on_opcode_f32(WasmBinaryReaderContext* ctx,
+                                float value) {
+  Context* context = ctx->user_data;
+  size_t immediate_len = ctx->offset - context->current_opcode_offset;
+  log_opcode(context, ctx->data, immediate_len, "%f", value);
```

Comment:
maybe use %g to get nicer-looking values here (or "%.19g" to get full precision; not sure if we care about that here).